grpc-ballerina/tests/02_array_field_type_service_pb.bal

```diff
@@ -15,7 +15,7 @@
 // under the License.
 // This is server implementation for bidirectional streaming scenario
 
-public client class HelloWorld3Client {
+public isolated client class HelloWorld3Client {
     *AbstractClientEndpoint;
 
     private Client grpcClient;
```

Comment:
Shall we make this final? we need to change it in the tool as well.

Reply:
Updated in 8f72bf5